nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/StandardFlowSynchronizer.java

```diff
@@ -990,23 +1015,6 @@ private void updateProcessor(final ProcessorNode procNode, final ProcessorDTO pr
         }
 
         procNode.setAnnotationData(config.getAnnotationData());
-
-        if (config.getAutoTerminatedRelationships() != null) {
-            final Set<Relationship> relationships = new HashSet<>();
-            for (final String rel : config.getAutoTerminatedRelationships()) {
-                relationships.add(procNode.getRelationship(rel));
-            }
-            procNode.setAutoTerminatedRelationships(relationships);
-        }
-
-        procNode.setProperties(config.getProperties());
-
-        final ScheduledState scheduledState = ScheduledState.valueOf(processorDTO.getState());
```

Comment:
Not sure that I understand why this section of code was removed... if we call updateProcessor(), we still want to start/stop/disable/enable processors, no?